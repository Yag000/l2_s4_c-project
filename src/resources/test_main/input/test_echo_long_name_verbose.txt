echo 1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-12345678901234567890-1234567890
echo py
echo this_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testth
exit
