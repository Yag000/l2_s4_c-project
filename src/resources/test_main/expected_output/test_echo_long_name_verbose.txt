Parsing file src/resources/test_main/input/test_echo_long_name_verbose.txt ...
/$ echo 1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-12345678901234567890-1234567890
1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-1234567890-12345678901234567890-1234567890
/$ echo py
py
/$ echo this_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testth
this_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testthis_is_a_testth
/$ exit
